OfficeDev/Microsoft-Teams-Samples
# [Graph][RSC][Sample][New] - Create sample which shows use of RSC to call Graph APIs using C#

## Description

Create new sample code to demo [RSC: Resource-specific consent in Teams - Teams | Microsoft Docs...](https://docs.microsoft.com/en-us/microsoftteams/platform/graph-api/rsc/resource-specific-consent)

## Proposed Changes
1. Added a Tab Application to call Graph API
2. Tab Display the list of RSC permission for Channel Graph API
3. Created Read Me file

## Testing
1. Sideload manifest having RSC configured in Teams
![image](https://user-images.githubusercontent.com/50989436/105956626-18aca900-609e-11eb-8599-b42a08e4ff2f.png)

2. Select RSC consent to add as a Tab
![image](https://user-images.githubusercontent.com/50989436/105956706-3417b400-609e-11eb-82bf-60cfd4b83fa0.png)

3. See the Configured RSC Permissions for channel
![image](https://raw.githubusercontent.com/OfficeDev/Microsoft-Teams-Samples/v-trkum/RSCGraphAPI/samples/graph-rsc/csharp/RSCDemo/Images/Picture2.Png)
